packages/main/src/features/InputElementsFormSupport.ts

```diff
@@ -10,7 +10,8 @@ interface IFormElement extends UI5Element {
 	checked?: boolean,
 }
 
-type NativeInputUpdateCallback = (element: IFormElement, nativeInput: HTMLInputElement) => void;
+type NativeFormElement = HTMLInputElement & HTMLTextAreaElement;
```

Comment:
Isn't it more correct this type to be: `HTMLInputElement | HTMLTextAreaElement`?

Reply:
I started like this, but I we will need to make a lot of casts to HTMLInputElement in the components using form support.